Django 1.8

kostko opened the pull request.

kostko committed:
Migrated to Django 1.8.

kostko committed:
Fixed incorrect query.

kostko committed:
PEP8 fixes.

mitar commented on requirements-readthedocs.txt:
```diff
@@ -6,21 +6,20 @@ Sphinx==1.2.3
 django_polymorphic==0.7.1
-git+https://github.com/wlanslovenija/django-registration.git@670779dce3b81fabc7cc2394db1ddd45ffcc4dc7#egg=django_registration-wlanslovenija
-django-sekizai==0.8.1
+git+https://github.com/ubernostrum/django-registration.git@6a953b0044a7e82fca0f9ce3b7146a0dffec3a51#egg=django_registration-master
```
Ohhh. I think I have changed `django-registration` quite a bit from the original package. Not sure if just switching a package can really work. Please verify this. I assume we will have to update the whole registration code anyway. To custom Django user model, registration, login and logout notifications based on official signals, and even OAuth support (both as a consumer and service).

kostko replied:
Yes I know, but I had to update it as the used version was so old it didn't work anymore on Django 1.8.

kostko replied:
I've ported the things over to the new version, which uses generic views instead of the previously used "backend" class. There were some other fixes required due to changes in the authentication models and forms in Django 1.8.

kostko committed:
Disabling -Wall as it currently produces too much output.

This is a temporary change until django-polymorphic fixes the
deprecation warnings which otherwise cause travis to abort the
build due to a huge log output.

kostko committed:
Account module fixes.

kostko committed:
Minor fixes.
